tokenid.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2019-2021 Weald Technology Trading
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ens
+
+import (
+	"fmt"
+	"github.com/ethereum/go-ethereum/common/math"
+)
+
+// DeriveTokenIdFromENSDomain derive token_id from the ENS domain
```

Comment:
Would be good to have a bit more information about what the token ID is in the comment (specifically talk about it being the ERC-721 token ID).